Sources/Kitura/bodyParser/BodyParser.swift

```diff
@@ -60,11 +68,10 @@ public class BodyParser: RouterMiddleware {
             return next() // the body was already parsed
         }
 
-        guard request.headers["Content-Length"] != nil,
-            let contentType = request.headers["Content-Type"] else {
-                return next()
+        guard request.headers["Content-Length"] != nil else {
```

Comment:
Can't this just be `guard let contentType = request.headers["Content-Type"] else { }`?

Reply:
So if request.headers["Content-Type"] is allowed to be nil is can't be in a guard let. It have been moved down and made a normal let.
At the moment, request.headers["Content-Length"] is not allowed to be nil or we return. Would we want this header to also be allowed to be nil?